docs/docs/ecosystem/projects.md

```diff
@@ -30,10 +30,14 @@ Our products solve the simplest use case and give you the ability to instantly
 ready the system for more complex scenarios without painful and slow upgrade
 processes.
 
-The above-mentioned _identity management server_ is to be announced soon.
+The identity management server **ORY Kratos** enables you to implement login and
+registration in a secure and straightforward way.  
```

Comment:
Let's add "mfa / 2fa, profile management, session management, user management" to that list